src/modules/Elsa.JavaScript/Services/JintJavaScriptEvaluator.cs

```diff
@@ -171,11 +171,11 @@ private static void CreateVariableAccessors(Engine engine, ExpressionExecutionCo
         }
     }
 
-    private static void CreateActivityOutputAccessors(Engine engine, ExpressionExecutionContext context)
+    private static async Task CreateActivityOutputAccessors(Engine engine, ExpressionExecutionContext context)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.